pkg/syncthing/syncthing.go

```diff
@@ -394,7 +393,7 @@ func (s *Syncthing) ResetDatabase(ctx context.Context, dev *model.Dev, local boo
 		if err != nil {
 			log.Infof("error posting 'rest/system/reset' local=%t syncthing API: %s", local, err)
 			if strings.Contains(err.Error(), "Client.Timeout") {
-				return errors.ErrUnknownSyncError
+				return fmt.Errorf("error resetting syncthing database local=%t: %s", local, err.Error())
```

Comment:
the  'local=' makes parsing the error harder. can we instead have a function that returns a message with the "local" or "remote" as part of the message?
```suggestion
				return fmt.Errorf("failed to reset syncthing database local=%t: %s", local, err.Error())
```

Reply:
we follow this pattern in several places. would you mind opening an issue to track the change?